test/dlc_tests/benchmark/ec2/tensorflow/inference/test_performance_tensorflow_inference.py

```diff
@@ -63,50 +63,49 @@ def test_performance_ec2_tensorflow_inference_graviton_cpu(
 
 def ec2_performance_tensorflow_inference(image_uri, processor, ec2_connection, ec2_instance_ami, region, threshold):
     docker_cmd = "nvidia-docker" if processor == "gpu" else "docker"
+
+    # active python env location used with graviton AMI is different than x86_64 AMI
+    # using only "python3" on graviton will not yeiled the python env where tensorflow is installed
+    python_cmd = "/usr/bin/python3" if  "graviton" in image_uri else "python3"
+
     container_test_local_dir = os.path.join("$HOME", "container_tests")
     tf_version = "1" if is_tf_version("1", image_uri) else "2"
     _, tf_api_version = get_framework_and_version_from_tag(image_uri)
 
     num_iterations = 500 if is_pr_context() else 1000
     # Make sure we are logged into ECR so we can pull the image
     ec2_connection.run(f"$(aws ecr get-login --no-include-email --region {region})", hide=True)
-
     ec2_connection.run(f"{docker_cmd} pull -q {image_uri} ")
-
-    # Run performance inference command, display benchmark results to console
-
     if "graviton" in image_uri:
         # TF training binary is used that is compatible for graviton instance type
-
         ec2_connection.run(
             (
-                f"sudo apt install python3-pip"
+                f"/usr/bin/pip3 install --user --upgrade awscli boto3 grpcio"
```

Comment:
nit: maybe just leave a comment here saying the graviton AMI doesn't have these pkgs but the test needs it

Reply:
Understand the thought here. Am more looking at getting the AMIs consistent and removing most of this in the end. Better to look at refactoring this test on another PR as this PR is needed for patching of the DLC image. Thank you for the comment.